### DISCUSSION TITLE
Shovel included at "tools" config

### DISCUSSION THREAD

**Discussion by itsMarkiS** [PurpurMC/Purpur#618] [__Dismissed__] [answered]
well, there are tillables with hoes and strippable with axes, and we can add and customize which blocks should be affected and what will they transform into...
we could have this for shovels as well, since there are only the **shovel-turns-block-to-grass-path** option for shovel

**Comment by BillyGalbreath** [comment, marked as answer]
Already being worked on.

https://github.com/pl3xgaming/Purpur/discussions/449
